ui/app/src/components/DeleteDialog/DeleteDialogContainer.tsx

```diff
@@ -69,6 +69,12 @@ export function DeleteDialogContainer(props: DeleteDialogContainerProps) {
   const [confirmChecked, setConfirmChecked] = useState(false);
   const authoringBase = useSelection((state) => state.env.authoringBase);
 
+  useEffect(() => {
+    if (error) {
+      setApiState({ error });
+    }
+  }, [error, setApiState]);
+
```

Comment:
Why would you do this instead of just using the error prop directly?

Reply:
We handle two different errors in this dialog, the one when submitting, and the one (the newly added) when the `get_delete_package` fails. Before the change, when submitting we call the submit service and if it fails, we do `setApiState({ error })`. So now that I added the `get_delete_package` error I set the error in the same apiState to use the same mechanism